catroid/src/main/java/org/catrobat/catroid/content/bricks/FormulaBrick.java

```diff
@@ -111,7 +112,11 @@ public View getView(Context context) {
 		super.getView(context);
 		for (BiMap.Entry<FormulaField, Integer> entry : brickFieldToTextViewIdMap.entrySet()) {
 			TextView formulaFieldView = view.findViewById(entry.getValue());
-			formulaFieldView.setText(getFormulaWithBrickField(entry.getKey()).clone().getTrimmedFormulaString(context));
+			String text =
+					getFormulaWithBrickField(entry.getKey()).clone().getTrimmedFormulaString(context);
+			formulaFieldView.setText(
+					FormulaSpannableStringBuilder.INSTANCE.buildSpannableFormulaString(view.getContext(), text,
```

Comment:
remove '.INSTANCE', you will see the reason in the FormulaSpannableStringBuilder!